templates/chat/order-activity-card.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="dnd5e2 chat-card activation-card">
+
+    {{!-- Description --}}
+    <p class="order-description">{{{ description }}}</p>
```

Comment:
I wonder if we could sneak the order icon onto the card somehow, maybe large and mostly transparent on the background? Not a priority.

Reply:
I like it. Will try to get to it